CODEOWNERS

```diff
@@ -49,12 +49,12 @@
 /doc/cprover-manual/contracts* @tautschnig @feliperodri @SaswatPadhi
 /src/goto-diff/ @tautschnig @peterschrammel
 /src/jsil/ @kroening @tautschnig
-/src/memory-analyzer/ @tautschnig @chrisr-diffblue
+/src/memory-analyzer/ @tautschnig @chrisr-diffblue @peterschrammel
 /jbmc/src/jbmc/ @peterschrammel @romainbrenguier
 /jbmc/src/janalyzer/ @peterschrammel @romainbrenguier
-/jbmc/src/jdiff/ @peterschrammel
+/jbmc/src/jdiff/ @peterschrammel @chrisr-diffblue
 /src/cpp/ @kroening @tautschnig @peterschrammel
-/src/solvers/smt2 @kroening @martin-cs @tautschnig @peterschrammel @allredj @romainbrenguier
+/src/solvers/smt2 @kroening @martin-cs @tautschnig @peterschrammel @thomasspriggs @NlightNFotis @TGWDB
```

Comment:
:+1:

Reply:
This, I've worked on a bit in the past, however given that the aim is sunsetting it in favour of the new SMT backend, I'm not sure what value code-ownership of this brings (I expect this to be a low traffic area of the code, but then again I might be mistaken).

Reply:
I'm tempted to keep this since we've had a few fixes and updates here, and with work on the new SMT side I can see some overlap in the future (bug fixes, minor refactors, etc.).